CMakeLists.txt

```diff
@@ -160,6 +160,7 @@ endif()
 #Use add_definitions for now for older cmake versions
 cmake_policy(SET CMP0005 NEW)
 add_definitions(-DHIPSYCL_DEBUG_LEVEL=${HIPSYCL_DEBUG_LEVEL})
+option(HIPSYCL_NO_DEVICE_MANGLER "Disable the use of DeviceMangler. necessary for LLVM 13 distributed with the rocm packages." OFF)
```

Comment:
```suggestion
option(HIPSYCL_NO_DEVICE_MANGLER "Disables the use of clang 13+ device mangling support. This removes support for explicit multipass. Necessary when building hipSYCL against LLVM 13 distributed with ROCm 4.5.x packages. This should only be enabled if hipSYCL cannot be built otherwise!" OFF)
```